src/models.ts

```diff
@@ -0,0 +1,32 @@
+export interface JsonRpcRequest {
+  jsonrpc: string;
+  id: string;
+  method: string;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  params: Array<any>;
```

Comment:
I like the "no-explicit-any" lint. Here, I would slightly prefer:

```suggestion
  params: Array<unknown>;
```

`unknown` indicates any type, but that you don't know what it is (so you aren't allowed to use it without an explicit cast, instead of being able to use it however you want with `any`).